pkg/virt-controller/services/template.go

```diff
@@ -470,6 +470,11 @@ func (t *templateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, i
 
 	hostName := dns.SanitizeHostname(vmi)
 	enableServiceLinks := false
+
+	var seccompProfile *k8sv1.SeccompProfile = nil
+	if t.clusterConfig.GetConfig().SeccompConfiguration != nil {
+		seccompProfile = t.clusterConfig.GetConfig().SeccompConfiguration.LauncherPolicy
```

Comment:
If I set `installProfile` to true, would then the kubevirt profile be picked up? If not, would it make sense to automatically set the custom profile path on the pod if no specific launcher profile is specified?

I am not sure if there is value in separating these.


How about in general restructuring the api a little bit and allow a configuration like this:


Install and use the kubevirt-shipped profile (would also be configured on the pod automatically correctly):

````yaml
seccompConfiguration:
  virtualMachineInstanceProfile:
    KubeVirtProfile: {}
````

Or, as a mutually exclusive option:

````yaml
seccompConfiguration:
  virtualMachineInstanceProfile:
    CustomProfile:
      localhostProfile: /my/path/to/profile
````

Reply:
I like the suggested API. Reworking...